DOCKER.md

```diff
@@ -0,0 +1,34 @@
+# Viewing the AtoM docs with Docker Compose
+
+This [Access to Memory](https://www.accesstomemory.org/) documentation can be
+generated and viewed locally in a web browser with the provided Docker Compose
+configuration.  Viewing the documentation locally can be very helpful when
+making updates to quickly check for syntax or content errors.
+
+## Prerequistes
+
+**These instructions require a working installation of the Docker Engine**
+
+See [Docker Engine Install](https://docs.docker.com/engine/install/) for
+instructions on installing Docker in your operating system.
+
+## Building a viewing the documentation with Docker Compose
+
+1. In the atom-docs directory run `docker-compose up -d` to build and start
+   the atom-docs Docker container
+1. Once the container is built, open a web browser and navigate to
+  `http://localhost:8100`
+
+![Screenshot of AtoM docs in Docker](images/atom_docs_in_docker.png)
+
+## Viewing updates
+
+The docker container uses
+[Sphinx Autobuild](https://pypi.org/project/sphinx-autobuild/) to automatically
+rebuild the Sphinx documents when any files are changed in the atom-docs
+directory.  After saving your changes the web documentation should automatically
+update in your browser.
+
+## When you are done
+
+In the atom-docs directory run `docker-compose stop` to stop the container
```

Comment:
Having only one container, I don't know if `down` would be better than `stop` or if it has other implications.

Reply:
Huh, I've been using `stop` for my AtoM docker-compose env, but it sounds like `down` removes the containers and networks in addition to stopping the container. 😎  Running `docker-compose up -d` after `down` doesn't seem noticeably slower than after `stop`, so I agree that `down` is probably better.